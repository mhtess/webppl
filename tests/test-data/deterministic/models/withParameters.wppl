randomIntegerERP.withParameters([2]).toJSON()
