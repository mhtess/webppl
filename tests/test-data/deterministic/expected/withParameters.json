{
  "result": {
    "probs": [0.5, 0.5],
    "support": [0, 1]
  }
}
